python/oneflow/test/modules/test_ctc_loss.py

```diff
@@ -323,6 +337,14 @@ def test_ctc_loss_with_diff_device_input(test_case):
         for arg in GenArgList(arg_dict):
             _test_ctc_loss_with_diff_device_input(test_case, *arg)
 
+    
+    def test_ctc_loss_functional(test_case):
+        arg_dict = OrderedDict()
+        arg_dict["reduction"] = ["mean", "none"]
+        for arg in GenArgList(arg_dict):
+            _test_ctc_loss_functional(test_case, *arg)
```

Comment:
这里调用 GenArgList 是多此一举的八股文写法，可以改掉，包括 test_ctc_loss_with_diff_device_input 里的：

```suggestion
        for reduction in ['mean', 'none']:
            _test_ctc_loss_functional(test_case, reduction)
```

即使这样改了之后，这里的遍历 `reduction in ['mean', 'none']` 也是没有必要的写法，reduction 的枚举可以在 autotest 里更优雅的完成